content/how-to/cloud-backup/rackspace-cloud-backup-backup-actions/index.md

```diff
@@ -139,7 +139,7 @@ You can change the files and folders that are saved in your backup.
 You can prevent a backup from running by selecting **Disable Backup**
 from the **Actions** menu.
 
-{{<image src="ScreenShot2015-10-16at2.36.44PM.png" alt="" title="">}}
+{{<image src="ScreenShot2022-08-09at2.36.44PM.png" alt="" title="">}}
```

Comment:
renaming the image name to something related to the screen showing would help. Perhaps something like `configured-backups`. You can add a sequence number at the end if multiple screenshots exist for the same page or action. Example `configured-backups-01`. 
OR just title with the article name and a number at the end.

Reply:
resolved